lib/console_runner.rb

```diff
@@ -0,0 +1,27 @@
+module TicTacToe
+
+  class ConsoleRunner
+    TTT_MARKER1, TTT_MARKER2 = 'X', 'O'
+
+    def initialize(marker_loop, game_completion, console_ui)
+      @marker_loop = marker_loop
+      @game_completion = game_completion
+      params = {
+      :console_ui => console_ui,
+      :validation => Validation.new
+      }
+      @game_turn = GameTurn.new(params)
+      @game_eval = GameEval.new
+    end
+
+    def run 
```

Comment:
Another name that I've seen used for this type of method is `#execute`. Nothing to change here, `#run` is a completely valid name, but I thought I'd just share that here since naming can sometimes be tricky so I find it helpful to be aware of different naming options